README.md

```diff
@@ -4,7 +4,7 @@ Terraform that builds an EKK logging stack.
 
 This stack is based on [this CloudFormation example.](https://us-west-2.console.aws.amazon.com/cloudformation/designer/home?region=us-west-2&templateUrl=https://s3.amazonaws.com/scriptdepot/es.template)
 
-It currently does NOT introduce any demonstration data. Future revisions may include demo data or integration tests.
+The stack also creates a small EC2 instance (defined in ec2-test.tf) that will be configured with a kinesis agent to test writing into the stream. If you do not wish to deploy this instance, move this file out of the terraform directory or change the extension of the file.
```

Comment:
Can't we use a `test/` module for that, which pulls in the top-level module? That way the files don't need to be modified.

Also, maybe include some instructions about what a user/contributor should be looking for to check that it works?